### PR TITLE
Removed the 3rd party links and vendor specific information from networking.md

### PR DESCRIPTION
**Pull request by kalashjain23**
PR for the issue #30745

* Solved this issue to update the docs to match the <a href="https://kubernetes.io/docs/contribute/style/content-guide/#third-party-content">style of 3rd party documentation </a>according to Kubernetes.

* Removed some content [ about ACI, Coil, HybridNet, L2 networks and Linux bridging ] that was not required K8s to function and some 3rd party links (including links to 3rd party's GitHub repositories ). 

-> I might be very wrong about the removal of some of the content as this is my first time contributing to Kubernetes, so please inform me about the changes that are still expected from my side. Thank You! I will always be looking forward to contribute to this community..!!

### CODE REVIEW

**Comment by kalashjain23** [comment]
I signed it! (CLA)

File: content/en/docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -43,49 +43,44 @@ imply any preferential status.
...
 Thanks to the "programmable" characteristic of Open vSwitch, Antrea is able to implement an extensive set of networking and security features and services on top of Open vSwitch.
 
 ### AWS VPC CNI for Kubernetes
 
-The [AWS VPC CNI](https://github.com/aws/amazon-vpc-cni-k8s) offers integrated AWS Virtual Private Cloud (VPC) networking for Kubernetes clusters. This CNI plugin offers high throughput and availability, low latency, and minimal network jitter. Additionally, users can apply existing AWS VPC networking and security best practices for building Kubernetes clusters. This includes the ability to use VPC flow logs, VPC routing policies, and security groups for network traffic isolation.
+The AWS VPC CNI) offers integrated AWS Virtual Private Cloud (VPC) networking for Kubernetes clusters. This CNI plugin offers high throughput and availability, low latency, and minimal network jitter. Additionally, users can apply existing AWS VPC networking and security best practices for building Kubernetes clusters. This includes the ability to use VPC flow logs, VPC routing policies, and security groups for network traffic isolation.
```
Comment (jihoon-seo) [inline content/en/docs/concepts/cluster-administration/networking.md:53]:
```suggestion
The AWS VPC CNI offers integrated AWS Virtual Private Cloud (VPC) networking for Kubernetes clusters. This CNI plugin offers high throughput and availability, low latency, and minimal network jitter. Additionally, users can apply existing AWS VPC networking and security best practices for building Kubernetes clusters. This includes the ability to use VPC flow logs, VPC routing policies, and security groups for network traffic isolation.
```

**Review by jihoon-seo — COMMENTED** [review]
@kalashjain23 Thank you for your contribution! 😊

CC @jimangel @sftim (#30745)

In my opinion, **just** disabling hyperlinks might lower the convenience of the readers (who might want take looks into external pages).

And the suggestion proposed in #30745 might be somehow different from the changes this PR is proposing.

What do you guys think? 😊

And below is a small nit:

File: content/en/docs/concepts/cluster-administration/networking.md (hunk 1 above)
Comment (kalashjain23) [reply content/en/docs/concepts/cluster-administration/networking.md:53]: > In my opinion, just disabling hyperlinks might lower the convenience of the readers (who might want take looks into external pages).

To be honest, I felt the same when removing those links but was just confused what to do and what not to according to that issue, so just gave it a try.
I will be glad if you can guide me and give me some small hints about the changes that I should not have done and changes that I should have done :)
Also, thanks for that small nit XD
Thanks for reviewing 😊 @jihoon-seo 🙌

Comment (kalashjain23) [reply content/en/docs/concepts/cluster-administration/networking.md:53]: ss
Also, can you tell me about this conflict a little bit, like what is creating this conflict and what can I do about it.

**Comment by jihoon-seo** [comment]
@kalashjain23 

[`Contiv-VPP` section is added 17 days ago (on 2022-01-07)](https://github.com/kubernetes/website/blame/main/content/en/docs/concepts/cluster-administration/networking.md#L107),
so I guess that you created your working branch from the `main` branch which is older than 2022-01-07.

[How to resolve conflict]
1. Delete 3 lines
   - `<<<<<<< kuber`
   - `=======`
   - `>>>>>>> main`
2. Make sure that the blank line placing is the same as [here](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/cluster-administration/networking.md?plain=1#L100-L112),
and save the changed file.
3. Continue rebase

See: https://www.simplilearn.com/tutorials/git-tutorial/merge-conflicts-in-git and so on..

**Pull request closed without merging**